Jellyfin.Api/Controllers/MediaSegmentController.cs

```diff
@@ -0,0 +1,150 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+using Jellyfin.Data.Entities;
+using Jellyfin.Data.Enums;
+using MediaBrowser.Common.Api;
+using MediaBrowser.Model.MediaSegments;
+using MediaBrowser.Model.Querying;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Jellyfin.Api.Controllers;
+
+/// <summary>
+/// Media Segments controller.
+/// </summary>
+[Authorize]
+public class MediaSegmentController : BaseJellyfinApiController
```

Comment:
All endpoints use the QueryResult type and I'm honestly not sure why, it just wraps the list and adds two properties of which one isn't even useful for these endpoints and the other one is the length of the list.

Reply:
I wanted a plain array or object body (code was initially c/p from another controller and missed that ActionResult was everything i need)